scripts/testnet/include/tools.sh

```diff
@@ -6,7 +6,9 @@ startProxy() {
 
   setWorkdirForNextCommands "$TESTNETDIR/proxy"
   if [ "$SOVEREIGN_DEPLOY" -eq 1 ]; then
-      ARGS="--sovereign-config"
+      ARGS="--sovereign"
+  else
```

Comment:
I think we should define it **above the first if** `ARGS=""`
Then only change it `if [ "$SOVEREIGN_DEPLOY" -eq 1 ]`(and remove else)